pyproject.toml

```diff
@@ -1,24 +1,26 @@
 [tool.poetry]
 name = "sql_metadata"
 version = "2.8.0"
-license="MIT"
+license = "MIT"
```

Comment:
All these formatting changes in this file seem like changes we might not want if it makes it differ from upstream?

Reply:
sorry I think this was an overzealous vs code toml formatter - will revert the formatting changes